src/briefcase/__main__.py

```diff
@@ -16,12 +17,16 @@ def main():
         log.error()
         log.error(str(e))
         result = e.error_code
+        log.capture_stacktrace()
     except KeyboardInterrupt:
         log.warning()
         log.warning("Aborted by user.")
         log.warning()
         result = -42
+        if getattr(command, "save_log", False):
+            log.capture_stacktrace()
```

Comment:
The exception details are only maintained within the except block.....so they have to be gathered here.

Reply:
My thought process here is the stacktrace may be useful for `KeyboardInterrupt` because the stracktrace will contain the `locals` for each level of the stack. For instance, let's say the user is sending CTRL+C because a process is hanging and that's the only way for the user to regain control. In that situation, I could imagine knowing all `locals` thru the stack being useful despite the user actually causing the exit.